<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CANSET</class>
 <widget class="QWidget" name="CANSET">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>155</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>241</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CAN SET</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Footer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="font_page_pbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目  录</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_page_pbtn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QWidget" name="Baudrate" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QLabel" name="baudrate_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Baudrate:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_1000k_checkBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1000k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_800k_checkBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>800k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_500k_checkBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>500k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_250k_checkBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>250k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_125k_checkBox">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>125k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_100k_checkBox">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>60</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>100k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_50k_checkBox">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>50k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_20k_checkBox">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>20k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_10k_checkBox">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>10k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_5k_checkBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>5k</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="baudrate_2k_checkBox">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>120</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>2.5k</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Period" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>531</width>
      <height>151</height>
     </rect>
    </property>
    <widget class="QLabel" name="period_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Period:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="period_line">
     <property name="geometry">
      <rect>
       <x>153</x>
       <y>70</y>
       <width>120</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Send" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>350</y>
      <width>531</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QLabel" name="send_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>420</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>115</y>
       <width>420</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>420</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>205</y>
       <width>420</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>2</width>
       <height>135</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>2</width>
       <height>135</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_form_line_7">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>2</width>
       <height>135</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_value_display_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>50</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_value_0x_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>20</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_value_display_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>140</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_value_text_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>210</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SendSysValue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_sta_0x_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>115</y>
       <width>20</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_sta_text_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>115</y>
       <width>210</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SendSysSta</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_sta_display_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>115</y>
       <width>50</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_sys_sta_display_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>115</y>
       <width>140</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_pgm_run_time_text_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>210</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SendSysSta</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_pgm_run_time_display_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>160</y>
       <width>140</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_pgm_run_time_0x_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>20</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="send_pgm_run_time_display_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>50</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Set" native="true">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>350</y>
      <width>471</width>
      <height>201</height>
     </rect>
    </property>
    <widget class="QLabel" name="set_form_line_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>420</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="set_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="set_form_line_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>115</y>
       <width>420</width>
       <height>2</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="set_form_line_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>2</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="set_form_line_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>2</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="set_form_line_5">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>2</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="set_sys_value_display_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>50</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="set_sys_value_0x_label">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>20</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="set_sys_value_display_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>70</y>
       <width>140</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="set_sys_value_text_label">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>210</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SetSysValue</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
